src/dsp_tools/models/user.py

```diff
@@ -608,60 +596,52 @@ def update(self, requesterPassword: Optional[str] = None) -> Any:
         jsonobj = self.toJsonObj(Actions.Update)
         if jsonobj:
             jsondata = json.dumps(jsonobj)
-            result = self._con.put(User.IRI + quote_plus(self.id) + "/BasicUserInformation", jsondata)
```

Comment:
here and in the following occurrences, the result was not used. So it's better not to store it in a variable